plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -1753,22 +1782,8 @@ $.extend(DataTable.prototype, UIControl.prototype, {
         relatedReports.each(function () {
             var clicked = this;
             $(this).unbind('click').click(function (e) {
-                var url = $(this).attr('href');
```

Comment:
I noticed this code and removed it as it is no longer working and I think not needed.